common-messaging/src/main/scala/org/genivi/sota/messaging/kinesis/KinesisClient.scala

```diff
@@ -42,11 +42,11 @@ object KinesisClient {
       awsSecretKey <- awsConfig.readString("secretAccessKey")
     } yield new StaticCredentialsProvider(new BasicAWSCredentials(awsAccessKey, awsSecretKey))
 
-  private[this] def getStreamNameSuffix(streamNameRoot: String, config: Config): ConfigException Xor String =
+  private[this] def streamNameWithSuffix(config: Config): ConfigException Xor (String => String) =
     for {
-      awsConfig <- config.configAt("aws")
-      streamNameSuffix <- config.readString("streamNameSuffix")
-    } yield streamNameRoot + "-" + streamNameSuffix
+      awsConfig <- config.configAt("messaging.kinesis")
```

Comment:
I still get config key not found exceptions when testing this fix locally. Have you tested it?